src/Checksum_Core_Command.php

```diff
@@ -216,23 +203,21 @@ private static function find_var( $var_name, $code ) {
 	private static function get_core_checksums( $version, $locale ) {
 		$url = 'https://api.wordpress.org/core/checksums/1.0/?' . http_build_query( compact( 'version', 'locale' ), null, '&' );
 
-		$options = array(
-			'timeout' => 30
-		);
+		$options = [ 'timeout' => 30 ];
 
-		$headers = array(
-			'Accept' => 'application/json'
-		);
+		$headers  = [ 'Accept' => 'application/json' ];
 		$response = Utils\http_request( 'GET', $url, null, $headers, $options );
 
-		if ( ! $response->success || 200 != $response->status_code )
+		if ( ! $response->success || 200 !== intval( $response->status_code ) ) {
```

Comment:
Please use a type cast. `intval()` is PHP 4-style code and nowadays only really intended for callbacks.
```suggestion
		if ( ! $response->success || 200 !== (int) $response->status_code ) {
```